### DISCUSSION TITLE
Unable to install fastapi-users

### DISCUSSION THREAD

**Discussion by jasonvriends** [fastapi-users/fastapi-users#803] [General]
Trying to install fastapi-users yields this error.

ERROR: Could not find a version that satisfies the requirement fastapi-users (from versions: none)
ERROR: No matching distribution found for fastapi-users

**Comment by frankie567** [comment]
A few thoughts:
* Are you making request again the public PyPi repository?
* Are you working with Python >=3.7? 
* Did you try the same on a clean virtual environment? If it's working, maybe you have dependency conflicts inside your project.

**Reply by jasonvriends** [reply]
Brand new VM
Everything default
Python 3.6.8
Occurs on all projects
Running pipenv install fast-api with no other params

**Reply by frankie567** [reply]
FastAPI Users only support Python >= 3.7; that's why pip can't install it from Python 3.6.

**Reply by jasonvriends** [reply]
This is excellent thank you. Python 3.6 is the default for AlmaLinux.
